src/app/docs/numeric/numeric-docs.component.html

```diff
@@ -6,4 +6,199 @@
   <sky-docs-demo-page-summary>
     The numeric pipe and numeric service format numbers with proper localization and currency symbol placement.
   </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <sky-definition-list
+      labelWidth="200px"
+    >
+      <sky-definition-list-heading>
+        This is a sample of numeric formatting.
+      </sky-definition-list-heading>
+      <sky-definition-list-content>
+        <p>
+          <sky-definition-list-label>
+            1 with zero digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1 | skyNumeric:{digits: 0} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1000 with zero digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1000 | skyNumeric:{digits: 0} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1000 with one digit:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1000 | skyNumeric}}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1100 with zero digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1100 | skyNumeric:{digits: 0} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1100 with one digit:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1100 | skyNumeric}}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1500 with zero digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1500 | skyNumeric:{digits: 0} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1500 with one digit:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1500 | skyNumeric}}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1000000 with any digits:
```

Comment:
Why does this say `any digits`? Shouldn't it be `zero digits`? It looks like `0` in the code.

Reply:
Fixed.